toolkit/types/cvss/cvss_v2_parse.rl

```diff
@@ -53,9 +53,11 @@ cr  = ( "CR"  >metric_start %mark_metric) ":" ( ("L"|"M"|"H"|"ND")           >va
 ir  = ( "IR"  >metric_start %mark_metric) ":" ( ("L"|"M"|"H"|"ND")           >value_start %mark_value);
 ar  = ( "AR"  >metric_start %mark_metric) ":" ( ("L"|"M"|"H"|"ND")           >value_start %mark_value);
 
-metric = ( av | ac | au | c | i | a | e | rl | rc | cdp | td | cr | ir | ar );
+base = av "/" ac "/" au "/" c "/" i "/" a;
+temporal = e "/" rl "/" rc;
+environmental = cdp "/" td "/" cr "/" ir "/" ar;
```

Comment:
I'm not as familiar with these. Let's say `E` is set. Does that mean `RL` and `RC` both must also be set?

Reply:
I think so, yes. The v2 spec is (IMO) unclear on this, but seems to say that if you use a metric group, all the metrics must be filled out.